CHANGELOG.md

```diff
@@ -4,7 +4,15 @@ This file is used to list changes made in each version of the logrotate cookbook
 
 ## Unreleased
 
-- resolved cookstyle error: attributes/default.rb:32:43 refactor: `ChefRedundantCode/MultiplePlatformChecks`
+### Changed
```

Comment:
We're not doing the subsections in the changelog anymore, can you please remove this?